### DISCUSSION TITLE
Question about how to obtain the initial model

### DISCUSSION THREAD

**Discussion by dahui-y** [nextpyp/.github#134] [Q&A]
Hi!

In step 6, Reference-based refinement, we need to input an initial model for refinement. I am wondering how we can obtain the initial model if we process another dataset. I am trying to reproduce the result of EMPIAR-10499. I am wondering if the initial model is the result of subtomogram  averaging, which we can also find on the EMPIAR website.

Thanks

******************************************************************************************************************************************
17 Jan Updated

Extend this question: if we would like to investigate things with unknown structures in cellular, which cannot be found in the EMDB, how do we obtain the initial reference model?

**Comment by abartesaghi** [comment]
Yes, you can use that as a reference. You just need to make sure that the pixel size in the header of the .mrc file is set correctly (which I believe should be case for maps downloaded from the EMDB).

**Reply by dahui-y** [reply]
Thank you for your reply, Prof. Abartesaghi! Extend this question: if we would like to investigate things with unknown structures in cellular, which cannot be found in the EMDB, how do we obtain the initial reference model?

**Reply by abartesaghi** [reply]
If you don't have a reference, you'll need to perform ab-initio refinement. Unfortunately, the current version of nextPYP doesn't support ab-initio refinement. However, you can export sub-volumes from the `Sub-tomogram extraction` tab in the Pre-processing block and use them to generate an initial model with standard sub-volume averaging tools. Once you're satisfied with the model, you can return to nextPYP to perform reference-based refinement. Alternatively, depending on the number of particles you have and the complexity of your target, you might try running several iterations of classification without alignment to see if any useful models emerge. Even if these are low resolution, they could serve as a starting point for reference-based refinement.
